.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           python-version: "3.11"
       - uses: astral-sh/setup-uv@v5
       - run: |
-          uvx copier copy \
+          uvx --with jinja2-time --with arrow copier copy \
```

Comment:
That means we need to update the documentation (README instructions) to explain these new dependencies